src/main/java/htsjdk/samtools/SAMRecordUtil.java

```diff
@@ -39,7 +39,9 @@
  */
 @Deprecated
```

Comment:
I agree with @pshapiro4broad about using the javadoc tag.  We try to use it other places we deprecated things.